tutorials/modelBorgifier/driveModelBorgifier.m

```diff
@@ -0,0 +1,80 @@
+% This file is published under Creative Commons BY-NC-SA.
```

Comment:
I am not sure whether `driveModelBorgifier.m` is correctly placed here. Is this file even necessary?

May you generate `.html`, `.pdf`, and `.m` from the `.mlx` file?